collectors/jiraffe/collectors.py

```diff
@@ -296,25 +300,31 @@ def collect(self):
         return f"{self.name} is updated until {start_dt}: {len(project_fields)} Jira projects' metadata fetched"
 
     @transaction.atomic
-    def update_metadata(self, project_fields):
+    def update_metadata(
+        self, project_fields, projects_to_delete: Optional[list] = None
+    ):
         """
         remove old and store new Jira projects' metadata
         inside an atomic transaction
         """
-        JiraProjectFields.objects.all().delete()
+        if projects_to_delete is None:
+            projects_to_delete = []
```

Comment:
`projects_to_delete` is always a `list`, so what about making it an default value to skip this conversion?

Reply:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html#anti-pattern